src/SFA.DAS.Roatp.CourseManagement.Web/Models/ProviderCourseLocations/ProviderCourseLocationViewModel.cs

```diff
@@ -0,0 +1,41 @@
+﻿using SFA.DAS.Roatp.CourseManagement.Domain.ApiModels;
+
+namespace SFA.DAS.Roatp.CourseManagement.Web.Models.ProviderCourseLocations
+{
+    public class ProviderCourseLocationViewModel
+    {
+        public string LocationName { get; set; }
+        public LocationType LocationType { get; set; }
+        public bool? HasDayReleaseDeliveryOption { get; set; }
+        public bool? HasBlockReleaseDeliveryOption { get; set; }
+        public bool? OffersPortableFlexiJob { get; set; }
+        public string DeliveryOption()
+        {
+            if ((HasDayReleaseDeliveryOption.HasValue && HasDayReleaseDeliveryOption.Value) && (HasBlockReleaseDeliveryOption.HasValue && HasBlockReleaseDeliveryOption.Value))
+            {
+                return "Day & block release";
+            }
+            if (HasDayReleaseDeliveryOption.HasValue && HasDayReleaseDeliveryOption.Value)
+            {
+                return "Day release";
+            }
+            if (HasBlockReleaseDeliveryOption.HasValue && HasBlockReleaseDeliveryOption.Value)
+            {
+                return "Block release";
+            }
+            return string.Empty;
+        }
+        public string HasOffersPortableFlexiJob => OffersPortableFlexiJob.HasValue && OffersPortableFlexiJob.Value ? "Yes" : "No";
```

Comment:
I'd prefer the 'Yes'/'No' to be in a referable const or static class, to aid testing and make in more flexible if the words ever are required to get changed.